scripts/version

```diff
@@ -64,6 +64,13 @@ if [ "$AGENT_TAG" = dev ]; then
     AGENT_TAG="master-head"
 fi
 
+# If you’re changing this, also change rancher/cmd/rancherd/go.mod
+# Also, when changing RKE2_VERSION here, be sure to pull the variables that follow from https://github.com/rancher/rke2/blob/$RKE2_VERSION/scripts/version.sh
+RKE2_VERSION=v1.20.5-rke2r1
+ETCD_VERSION=${ETCD_VERSION:-v3.4.13-k3s1}
+PAUSE_VERSION=${PAUSE_VERSION:-3.2}
+IMAGE_BUILD_VERSION=${IMAGE_BUILD_VERSION:-build20210223}
```

Comment:
This script isn't just for RKE2/rancherd. These should all start with `RKE2_` or something so we differentiate, otherwise we can never have a rancher related `PAUSE_VERSION`

Reply:
Done.